lib/abci/handlers/query/identityQueryHandlerFactory.js

```diff
@@ -9,26 +9,44 @@ const NotFoundAbciError = require('../../errors/NotFoundAbciError');
 /**
  *
  * @param {IdentityStoreRepository} identityRepository
+ * @param {RootTree} rootTree
+ * @param {IdentitiesStoreRootTreeLeaf} identitiesStoreRootTreeLeaf
  * @return {identityQueryHandler}
  */
-function identityQueryHandlerFactory(identityRepository) {
+function identityQueryHandlerFactory(
+  identityRepository,
+  rootTree,
+  identitiesStoreRootTreeLeaf,
+) {
   /**
    * @typedef identityQueryHandler
    * @param {Object} params
    * @param {Object} options
    * @param {Buffer} options.id
+   * @param {Object} request
+   * @param {boolean} [request.prove]
    * @return {Promise<ResponseQuery>}
    */
-  async function identityQueryHandler(params, { id }) {
+  async function identityQueryHandler(params, { id }, request) {
+    const includeProof = request.prove === 'true';
```

Comment:
Why this is a string?

Reply:
According to the docs, the request should look like this
`curl -X GET "https://rpc.cosmos.network/abci_query?path=%2Fa%2Fb%2Fc&data=IHAVENOIDEA&height=1&prove=true" -H "accept: application/json"`

Reply:
Make sense.